mkdocs.yml

```diff
@@ -16,7 +16,8 @@ pages:
 - ['docs/getting-started.md', 'Documentation', 'Getting Started']
 - ['docs/schema.md', 'Documentation', 'Working with table schemas']
 #- ['docs/usage.md', 'Documentation', 'Usage']
-- ['docs/models.md', 'Documentation', 'Models']
+- ['docs/models-definition.md', 'Documentation', 'Models - Definition']
+- ['docs/models-usage.md', 'Documentation', 'Models - Usage']
```

Comment:
Doesn't look like that will add 3 levels though. And our "Documentation" takes up a level